use/singularity.md

```diff
@@ -5,33 +5,33 @@ aliases:
   - /use/singularity/
 ---
 
-[Singularity](https://www.sylabs.io/guides/latest/user-guide/) is useful for running containers as an unprivileged user, especially in multi-user environments like High-Performance Computing clusters.
+[Singularity](https://docs.sylabs.io/guides/latest/user-guide/) is useful for running containers as an unprivileged user, especially in multi-user environments like High-Performance Computing clusters.
 Rocker images can be imported and run using Singularity, with optional custom password support.
 
+:::{.callout-note}
+In this guide, *Singularity* can refer to either [SingularityCE](https://sylabs.io/singularity/) or [Apptainer](https://apptainer.org/).
+While Apptainer is generally [compatible with Singularity commands and environment variables](https://apptainer.org/docs/user/latest/singularity_compatibility.html), Apptainer users may wish to replace the `singularity` command with `apptainer`, and `SINGULARITY` with `APPTAINER` in environment variables.
+:::
+
 ## Importing a Rocker Image
 
 Use the `singularity pull` command to import the desired Rocker image from Docker Hub into a (compressed, read-only) Singularity Image File:
 
 ```bash
-singularity pull docker://rocker/rstudio:4.2
+singularity pull docker://rocker/rstudio:4.4.2
```

Comment:
How about `rocker/rstudio:4`?

Reply:
The protocol has changed somewhat over the course of the course of rocker/rstudio:4, mainly to RStudio Server 
 updates (e.g., https://github.com/rocker-org/rocker-versioned2/issues/105). I think a more-precise tag is needed for reproducibility.

Also, the Slurm job script illustrates setting R_LIBS_USER to separate R packages installed from within the rocker/rstudio container from R packages installed from the host environment (which on an HPC cluster would typically be a Linux distro related to RHEL or SLES). While normally installed R packages would be compatible within patch releases of the same minor R version (e.g., `R_LIBS_USER=$HOME/R/rocker-rstudio/4.4`), it seems this separation should be more fine-grained for rocker/rstudio releases, as rocker/rstudio OS versions have changed between patch releases (e.g., rocker/rstudio:4.4.1 == Ubuntu 22.04, rocker/rstudio:4.4.2 == Ubuntu 24.04). Adding the patch version to the R_LIBS_USER value (as proposed in the PR) would avoid the scenario where, e.g., a previous user-installed R package was linked against a software library that no longer exists in newer container images.

Alternatively, as an optimization, I suppose the OS version could be added to R_LIBS_USER (e.g., `R_LIBS_USER=$HOME/R/rocker-rstudio/4.4-Ubuntu-24.04`, though set dynamically), allowing previously installed R packages for the same minor R version to be reused if built on the same OS version?

Reply:
I see, thanks.